actin/algo/actin_treatment_matcher_cohort

```diff
@@ -12,7 +12,8 @@ fi
 patient=$(echo ${sample} | head -c 12)
 clinical_json="$(locate_actin_cohort_clinical_directory)/${patient}.clinical.json"
 molecular_json="$(locate_actin_cohort_molecular_directory ${sample})/${sample}.molecular.json"
-treatment_database_directory="$(locate_actin_cohort_treatment_database_directory)"
+trial_database_directory="$(locate_actin_cohort_treatment_database_directory)"
```

Comment:
maybe an opportunity to correct the naming and call the database file "trial" instead of treatment?

Reply:
We'll leave this as-is after discussion, as the treatment->trial rename requires more thought.